spec.md

```diff
@@ -92,7 +93,7 @@ that do not understand them.
 
 Requests from the platform to the service broker MUST contain a header that declares the version number of the Service Broker API that the marketplace will use:
 
-`X-Broker-Api-Version: 2.11`
+`X-Broker-API-Version: 2.11`
```

Comment:
Opened https://github.com/openservicebrokerapi/servicebroker/pull/246 for this change